### PR TITLE
Document feature_spaces_metaspaces

### PR DESCRIPTION
**Pull request by t3chguy**
---
This change is marked as an *internal change* (Task), so will not be included in the changelog.

### CODE REVIEW

File: docs/labs.md [hunk 1]
```diff
@@ -148,4 +148,9 @@ development will provide opportunities for feedback.
...
 
 Note that this feature is currently under active development and therefore is
-entirely incomplete and may not work at all - it is not recommended for general use at this time.
\ No newline at end of file
+entirely incomplete and may not work at all - it is not recommended for general use at this time.
+
+## Metaspaces (`feature_spaces_metaspaces`) [In Development]
+
+Metaspaces are automatically populated spaces you can enable in your Space panel.
+By default, you'll have Home or All rooms, but you can opt in to a People, Favourites, and Orphans metaspace too.
```
Comment (ShadowJonathan) [inline docs/labs.md:156]:
```suggestion
By default, you'll have Home or All rooms, but you can opt in to a People, Favourites, and Other Rooms metaspace too.
```

I assume that via [`2c13035` (#7077)](https://github.com/matrix-org/matrix-react-sdk/pull/7077/commits/2c1303579ff324595a38d0ab301ccb1531cd1daf)(?)

Comment (t3chguy) [reply docs/labs.md:156]: Thanks!

File: element.io/develop/config.json [hunk 2]
```diff
@@ -56,5 +56,8 @@
     "posthog": {
         "projectApiKey": "phc_Jzsm6DTm6V2705zeU5dcNvQDlonOR68XvX2sh1sEOHO",
         "apiHost": "https://posthog.hss.element.io"
+    },
+    "features": {
+        "feature_spaces_metaspaces": true
```
Comment (jryans) [inline element.io/develop/config.json:61]: This will force it on with no UI path to disable, so we may want to be sure that is okay.

**Review by jryans — APPROVED** [review]

File: element.io/develop/config.json (hunk 2 above)
Comment (t3chguy) [reply element.io/develop/config.json:61]: After discussion it is okay for this specific lab, but we will need a way to have default-on labs via config.json without them being forced-on for IA1.3 and IA1.5

**Pull request merged**